Packs/WhisperGateCVE-2021-32648/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,35 @@
+#### Playbooks
+
+##### New: WhisperGate and HermeticWiper & CVE-2021-32648
+
+- New: - On January 14th, 2022, reports began on a malware operation dubbed "WhisperGate" targeting multiple -organizations in Ukraine.
+
+- On February 23, 2022, a new wiper malware known as "HermeticWiper" was disclosed by several cybersecurity researchers. The new wiper "HermeticWiper" was also being used against organizations in Ukraine.
+
+CVE-2021-32648 vulnerability has a CVSS score of 9.1 and was found in octobercms, which is a CMS platform based on the Laravel PHP Framework. 
+In affected versions of the october/system package an attacker can request an account password reset and then gain access to the account using a specially crafted request. 
+The issue has been patched in Build 472 and v1.1.5.
+
+**The playbook includes the following tasks:**
+- Collect related known indicators from Unit 42, CISA and Malware News blog.
+-  Search for possible vulnerable servers using Xpanse.
+- Indicators and exploitation patterns hunting using PAN-OS, Cortex XDR and SIEM products.
+- Block indicators automatically or manually.
+
+**Mitigations:**
+* October CMS security recommendations
+* Deploy YARA detection Rules.
+
+More information:
+[UNIT42 Blog - Ongoing Russia and Ukraine Cyber Conflict](https://unit42.paloaltonetworks.com/ukraine-cyber-conflict-cve-2021-32648-whispergate/)
+[Russia-Ukraine Cyberattacks: How to Protect Against Related Cyberthreats Including DDoS, HermeticWiper, Gamaredon and Website Defacement](https://unit42.paloaltonetworks.com/preparing-for-cyber-impact-russia-ukraine-crisis/)
+[Microsoft Blog](https://www.microsoft.com/security/blog/2022/01/15/destructive-malware-targeting-ukrainian-organizations/)
+[CVE-2021-32648 NVD](https://nvd.nist.gov/vuln/detail/CVE-2021-32648#vulnCurrentDescriptionTitle)
+
+Note: This is a beta playbook, which lets you implement and test pre-release software. Since the playbook is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
+##### WhisperGate and HermeticWiper & CVE-2021-32648
+- Replaced the **Block Indicators - Generic v2** sub-playbook with the new **Block Indicators - Generic v3** playbook. (Available from Cortex XSOAR 6.5.0).
+- Added the *AutoBlockIndicators* input. This input determines whether to block the indicators automatically or manually using the **Block Indicators - Generic v3** sub-playbook. (Available from Cortex XSOAR 6.5.0).
+- Added the *UserVerification* input. This input determines whether blocking IPs should be done with or without user verification.  The IPs blocking is done by using the **Block Indicators - Generic v3** sub-playbook. (Available from Cortex XSOAR 6.5.0).
+- Added configuration `fromversion: 6.0.0` and `toversion: 6.4.9`
```

Comment:
please delete the following lines:
- Added configuration `fromversion: 6.0.0` and `toversion: 6.4.9`
- Added configuration `fromversion: 6.5.0`